library/src/main/java/com/mapzen/android/OverlayManager.java

```diff
@@ -35,11 +35,13 @@
   private static final String NAME_POLYGON = "mz_default_polygon";
   private static final String NAME_MARKER = "mz_default_point";
   private static final String NAME_START_PIN = "mz_route_start";
-  private static final String NAME_END_PIN = "mz_route_stop";
+  private static final String NAME_END_PIN = "mz_route_destination";
   private static final String NAME_DROPPED_PIN = "mz_dropped_pin";
   private static final String NAME_SEARCH_RESULT_PIN = "mz_search_result";
   private static final String NAME_ROUTE_PIN = "mz_route_location";
   private static final String NAME_ROUTE_LINE = "mz_route_line";
+  private static final String NAME_TRANSIT_ROUTE_LINE = "mz_route_line_transit";
```

Comment:
Fixed in https://github.com/tangrams/bubble-wrap/commit/e9172d25ed9d5aff90c6e94f09b34e4f7755e505.